apps/storybook/src/forms/ButtonGroup.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { Button, ButtonGroup } from "@optiaxiom/react";
+import { IconArrowRight, IconDownload, IconPhoto } from "@tabler/icons-react";
+
+const meta: Meta<typeof ButtonGroup> = {};
```

Comment:
we still needed the `component` here
```suggestion
const meta: Meta<typeof ButtonGroup> = {
  component: ButtonGroup,
};
```

Reply:
addressed bhai